Xamarin.Forms.Platform.Android/AppCompat/TabbedPageRenderer.cs

```diff
@@ -10,19 +10,22 @@
 using Android.Support.Design.Widget;
 using Android.Support.V4.App;
 using Android.Support.V4.View;
+using AWidget = Android.Widget;
 using Android.Views;
 using Xamarin.Forms.Internals;
 using Xamarin.Forms.PlatformConfiguration.AndroidSpecific;
 
 namespace Xamarin.Forms.Platform.Android.AppCompat
 {
-	public class TabbedPageRenderer : VisualElementRenderer<TabbedPage>, TabLayout.IOnTabSelectedListener, ViewPager.IOnPageChangeListener, IManageFragments
+	public class TabbedPageRenderer : VisualElementRenderer<TabbedPage>, TabLayout.IOnTabSelectedListener, ViewPager.IOnPageChangeListener, IManageFragments, BottomNavigationView.IOnNavigationItemSelectedListener
```

Comment:
I'd suggest making modifications to OnPagePropertyChanged to handle when a page's title changes (e.g. via data binding). The current renderer already does this when the native element is a TabLayout. I think the code you'd need for the BottomNavigationView is something like


var page = (Page)sender;
var index = Element.Children.IndexOf(page);
var tab = _bottomNavigationView.Menu.GetItem(index);
tab.SetTitle(page.Title);